opencga-app/app/scripts/azure/arm/validate.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cd $(dirname "$0")
+
+set -e
+npm install armval && node node_modules/.bin/armval "**/azuredeploy.json"
```

Comment:
Should this clean up or install globally? Just thinking you'll end up with a node_modules in every ARM template directory and not sure if that's optimal or not. @martinpeck's suggestion of using npx might be a better solution.

Reply:
assuming the `npx` suggestion isn't a bad steer, this would presumably result in the .gitginore changes being no longer needed

Reply:
Sweet ... moving to npx as much better!